### PR TITLE
Add: p0-question-page

### PR DESCRIPTION
**Pull request by gauravsinhaweb**
question-page

### CODE REVIEW

File: pages/quiz.html [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    <main class="wrapper_main flex">
+      <ul class="question_list p-sm">
+        <li>
+          <div>
+            What happens in the beginning of "Halo 2" that ends the award ceremony?          </div>
+          <ul class="options text_cl">
+            <li class="option">
+              <button class="btn btn-option out-primary flex">
+                <span>Everyone gets their awards</span>
+              </button>
```
Comment (pankaj-prog) [inline pages/quiz.html:49]: You can directly use the text inside button element only, no need for adding extra element (span here)

File: pages/quiz.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+ 
+    </main>     <!-- modal  -->
+      <div id="modal" class="modal">
+        <div class="card modal_box">
+          <div class="flex ">
+              <p class="text_cl p-sm">are you sure you want to submit the quiz?</p>
+          </div>
+                <div class="text_cl wrapper_confirm flex">
+                   <button class="btn btn-cta">confirm</button>
+                   <button id="close-modal" class="btn out-danger">cancel</button>
```
Comment (rahulk31) [inline pages/quiz.html:191]: Suggestion to improvise the classname: "wrapper-confirm", as it contains both the confirm and cancel buttons. May be you can name as cta-wrapper. As both the buttons are call to action buttons.

**Review by pankaj-prog — COMMENTED** [review]
UI looks awesome.

**Review by rahulk31 — APPROVED** [review]
Overall code structure is great and UI is neat. Can be merged. 👍

File: pages/quiz.html (hunk 1 above)
Comment (gauravsinhaweb) [reply pages/quiz.html:49]: Sure. I'll do that.

File: pages/quiz.html (hunk 2 above)
Comment (gauravsinhaweb) [reply pages/quiz.html:191]: made the changes. Thanks!

**Review by pankaj-prog — APPROVED** [review]

**Pull request merged**